docs/usage/create.rst.inc

```diff
@@ -329,14 +329,14 @@ Reading backup data from stdin
 There are two methods to read from stdin. Either specify ``-`` as path and
 pipe directly to borg::
 
-    backup-vm --id myvm --stdout | borg create REPO::ARCHIVE -
+    backup-vm --id myvm --stdout | borg create --repo REPO ARCHIVE -
```

Comment:
this is a auto-generated file, you need to edit the respective .py file to change contents here.

see comment in line 1 of this file.

Reply:
did you see my comment?